test/e2e/nftables_test.go

```diff
@@ -88,3 +100,39 @@ var _ = Describe("Nftables", func() {
 		}
 	})
 })
+
+func AddPortsToNFTables(nftables []byte, extraNFTablesFile string) ([]byte, error) {
+	nftStr := string(nftables)
+
+	insertPoint := "# Logging and default drop"
+	if !strings.Contains(nftStr, insertPoint) {
+		return nftables, fmt.Errorf("insert point not found in nftables configuration")
+	}
+
+	/*
```

Comment:
When I talked about an example I meant something like the samples/custom-entries files we have, can be a file or somewhere in the top part of the test file